### PR TITLE
[#42628] Enable feature flag for storages module on pull preview

### PR DESCRIPTION
**Pull request by wielinde**
https://community.openproject.org/work_packages/42628

### CODE REVIEW

File: .github/workflows/pullpreview.yml [hunk 1]
```diff
@@ -38,3 +38,4 @@ jobs:
         AWS_ACCESS_KEY_ID: "${{ secrets.AWS_ACCESS_KEY_ID }}"
         AWS_SECRET_ACCESS_KEY: "${{ secrets.AWS_SECRET_ACCESS_KEY }}"
         AWS_REGION: eu-central-1
+        OPENPROJECT_FEATURE__STORAGES__MODULE__ACTIVE: true
```
Comment (wielinde) [inline .github/workflows/pullpreview.yml]: Hey @crohr, do you have any clue, why this doesn't enable the feature? Is that the wrong place to set an ENV variable for OpenProject that is specific for the PULL PREVIEW environments?

Comment (crohr) [reply .github/workflows/pullpreview.yml]: @wielinde env variables for runtime need to be added here: https://github.com/opf/openproject/blob/4d3a3f09978bdcaf81f667764213d5c6068ad4e5/.github/workflows/pullpreview.yml#L20

Comment (wielinde) [reply .github/workflows/pullpreview.yml]: Oh, of cause. Thank you!

**Review by cbliard — APPROVED** [review]
LGTM

**Pull request merged**